test/build_tests/isa_tests/main.c

```diff
@@ -6,9 +6,10 @@
 
 ELTYPE summation(ELTYPE * data, int length) {
     ELTYPE sum = 0;
-    for( int idx=0; idx<length/STRIDE; ++idx ) {
-        for( int jdx=0; jdx<STRIDE; ++jdx ) {
-            sum += data[idx + jdx];
+    int idx, jdx;
+    for( idx=0; idx<length/STRIDE; ++idx ) {
+        for( jdx=0; jdx<STRIDE; ++jdx ) {
+            sum += data[idx * STRIDE + jdx];
```

Comment:
Not a fan of C99? ;)

Reply:
Not me in particular, but a compiler I used at some point these days wasn't happy about the C99-style of the loop :smile: